Editor/Scripts/PackageUpdate/PackageUpdateHandler.cs

```diff
@@ -0,0 +1,55 @@
+using Unity.FilmInternalUtilities.Editor;
+using UnityEditor;
+using UnityEditor.PackageManager;
+using UnityEditor.SceneManagement;
+using Object = UnityEngine.Object;
+
+
+namespace Unity.MeshSync.Editor{
+
+[InitializeOnLoad]
+internal class PackageUpdateHandler {
+    static PackageUpdateHandler() {
+        Events.registeredPackages -= OnPackageRegistered;
+        Events.registeredPackages += OnPackageRegistered;
+
+        Events.registeringPackages -= OnPackageRegistering;
+        Events.registeringPackages += OnPackageRegistering;
+    }
+
+    private static void OnPackageRegistering(PackageRegistrationEventArgs obj) {
+        if (obj.changedFrom.FindPackage("com.unity.meshsync") == null)
+            return;
+
+        StopAllServers();
```

Comment:
This will have a side effect of stopping all servers when downgrading to 0.15.0 and bellow. We can check the version and show a warning about this to the user.